scripts/benchmarks_ci.py

```diff
@@ -140,6 +143,24 @@ def __is_valid_datetime(dt: str) -> str:
             "%%Y-%%m-%%dT%%H:%%M:%%SZ").'''
     )
 
+    def __is_valid_sdk_version(version:str) -> str:
+        try:
+            if version is None or re.search('\d\.\d+\.\d+', version):
+                return version
+            else:
+                raise ValueError
+        except ValueError:
+            raise ArgumentTypeError(
+                'Version "{}" is in the wrong format'.format(version))
+
+    parser.add_argument(
```

Comment:
This would not be needed after suggestions below, except for moving the validation code to `dotnet.py`